### ISSUE TITLE
Export to JSON/YAML; generate reports from JSON/YAML

### ISSUE THREAD

**Issue by kopf** [pivotal/LicenseFinder#491]
It would be great if something like the following were possible:

```
license_finder --format yaml . > /tmp/licenses.yaml
license_finder report --format html < /tmp/licenses.yaml > /tmp/licenses.html
license_finder report --format csv < /tmp/licenses.yaml > /tmp/licenses.csv
```

Basically: run the license checks *once*, output to a yaml (or json, or whatever) file, and generate the reports from this file. If `license_finder report` is run without input from STDIN, it would behave as usual. 

The motivation is that currently the license checking step on one of our projects takes about 7 minutes in order to generate both CSV and HTML outputs. If the licenses were checked just once, this time would be (at least) halved. 

I'm not a ruby pro, but if you were to tell me if the interface/workflow outlined above was acceptable for you, I could possibly look in to implementing it.

**Comment by mkoertgen** [comment]
I would love to have json output

**Comment by mdb** [comment]
Note that PR #500 adds JSON output. Looks like we're waiting on @iamjarvo to add tests to the PR.

**Comment by kdykeman** [comment]
For adding JSON support, PR #500 has been superseded by PR #521 which has been merged and should be in a new release shortly.
Is there still a desire for YAML support as well? If so, I'll close this issue and open a separate one for YAML support.

**Comment by kopf** [comment]
Cool! I'd be fine with JSON. 

Is there already a functionality for converting between formats? (e.g. JSON input, csv output)

If not, and since this issue really consisted of two features (yaml output, and conversion), I could either adjust this issue to just focus on conversion, or close this issue and create a new one for conversion. Just let me know.
